src/data/markdown/translated-guides/en/04 Results visualization/00 End-of-test Summary.md

```diff
@@ -3,102 +3,120 @@ title: 'End-of-test summary'
-     http_req_receiving.........: min=33.36µs  avg=2.66ms   med=180.36µs p(95)=2.4ms    p(99)=128.79ms p(99.99)=205.16ms max=205.45ms
-     http_req_sending...........: min=6.09µs   avg=44.92µs  med=35.77µs  p(95)=98.26µs  p(99)=148.49µs p(99.99)=1.09ms   max=5.53ms
-     http_req_tls_handshaking...: min=0s       avg=1.12ms   med=0s       p(95)=0s       p(99)=0s       p(99.99)=447.46ms max=614.35ms
-     http_req_waiting...........: min=131.3ms  avg=147.92ms med=137.57ms p(95)=267.49ms p(99)=282.23ms p(99.99)=982.55ms max=1.01s
-     http_reqs..................: 13416   44.111343/s
-     iteration_duration.........: min=2.28s    avg=3.83s    med=3.82s    p(95)=5.2s     p(99)=5.36s    p(99.99)=6.1s     max=6.18s
-     iterations.................: 3354    11.027836/s
-     vus........................: 1       min=1   max=50
-     vus_max....................: 50      min=50  max=50
+Ramp_Up ✓ [======================================] 00/20 VUs  30s
+     █ GET home - https://example.com/
+
+       ✓ status equals 200
+       
+     █ Create resource - https://example.com/create
+
+       ✗ status equals 201
+        ↳  0% — ✓ 0 / ✗ 45
+
+     checks.........................: 50.00% ✓ 45       ✗ 45  
+     data_received..................: 1.3 MB 31 kB/s
+     data_sent......................: 81 kB  2.0 kB/s
+     group_duration.................: avg=6.45s    min=4.01s    med=6.78s    max=10.15s   p(90)=9.29s    p(95)=9.32s   
+     http_req_blocked...............: avg=57.62ms  min=7µs      med=12.25µs  max=1.35s    p(90)=209.41ms p(95)=763.61ms
+     http_req_connecting............: avg=20.51ms  min=0s       med=0s       max=1.1s     p(90)=100.76ms p(95)=173.41ms
+   ✗ http_req_duration..............: avg=144.56ms min=104.11ms med=110.47ms max=1.14s    p(90)=203.54ms p(95)=215.95ms
+       { expected_response:true }...: avg=144.56ms min=104.11ms med=110.47ms max=1.14s    p(90)=203.54ms p(95)=215.95ms
+     http_req_failed................: 0.00%  ✓ 0        ✗ 180 
+     http_req_receiving.............: avg=663.96µs min=128.46µs med=759.82µs max=1.66ms   p(90)=1.3ms    p(95)=1.46ms  
+     http_req_sending...............: avg=88.01µs  min=43.07µs  med=78.03µs  max=318.81µs p(90)=133.15µs p(95)=158.3µs 
+     http_req_tls_handshaking.......: avg=29.25ms  min=0s       med=0s       max=458.71ms p(90)=108.31ms p(95)=222.46ms
+     http_req_waiting...............: avg=143.8ms  min=103.5ms  med=109.5ms  max=1.14s    p(90)=203.19ms p(95)=215.56ms
+     http_reqs......................: 180    4.36938/s
+     iteration_duration.............: avg=12.91s   min=12.53s   med=12.77s   max=14.35s   p(90)=13.36s   p(95)=13.37s  
+     iterations.....................: 45     1.092345/s
+     vus............................: 1      min=1      max=19
+     vus_max........................: 20     min=20     max=20
+
+ERRO[0044] some thresholds have failed  
 ```
 
 </CodeGroup>
 
-## Summary options
+<Collapsible title="Summary export to a JSON file (Discouraged)">
```

Comment:
I suggest moving the content of the Collapsible component to the Summary Options section. Something alongs the line:

- The [--summary-export:](/using-k6/options#summary-export) exports a summary report with a predefined JSON format to a file. This option is Discouraged. Use the `handleSummary` callback instead.


Note that the same Collapsible component is shown twice. The next one is in the `Custom output examples` section.  This is a duplication and should be removed.